tests/0081-admin.c

```diff
@@ -4009,7 +4105,9 @@ static void do_test_apis(rd_kafka_type_t cltype) {
                     rd_true /*with subscribing consumer*/, rd_true);
         }
 
-        do_test_ScramConfigAdmin(rk,mainq);
+        if (test_broker_version >= TEST_BRKVER(2, 7, 0, 0)) {
+                do_test_UserScramCredentials(rk, mainq);
```

Comment:
For parity with other tests add test with NULL queue (using temp queue), and a "what" field in the same way.